mbed_greentea/mbed_greentea_cli.py

```diff
@@ -281,6 +281,11 @@ def main():
                     default=None,
                     help='Shuffle seed (If you want to reproduce your shuffle order please use seed provided in test summary)')
 
+    parser.add_option('', '--sync',
+                    dest='sync_packet',
```

Comment:
This is not a good name for this parameter. `sync_num` or `num_sync_packtes` or something like that is a better name.